srcpkgs/linux5.5/files/x86_64-dotconfig

```diff
@@ -401,11 +400,10 @@ CONFIG_MICROCODE_AMD=y
 CONFIG_MICROCODE_OLD_INTERFACE=y
 CONFIG_X86_MSR=m
 CONFIG_X86_CPUID=m
-CONFIG_X86_5LEVEL=y
+# CONFIG_X86_5LEVEL is not set
```

Comment:
ditto

Reply:
We dont need this: "5-level paging enables access to larger address space: upto 128 PiB of virtual address space and 4 PiB of physical address space.